src/tests/ftest/io/ior_small.py

```diff
@@ -54,19 +55,24 @@ def test_ior_small(self):
         transfer_block_size = self.params.get("transfer_block_size",
                                               '/run/ior/iorflags/*')
         obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
-
+        hdf5_plugin_path = None
         # run tests for different variants
         self.ior_cmd.flags.update(flags[0])
         for oclass in obj_class:
             self.ior_cmd.dfs_oclass.update(oclass)
             for api in apis:
-                self.ior_cmd.api.update(api)
+                if api == "HDF5-VOL":
+                    self.ior_cmd.api.update("HDF5")
+                    hdf5_plugin_path = self.params.get(
+                        "plugin_path", '/run/hdf5_vol/*')
+                else:
+                    self.ior_cmd.api.update(api)
                 for test in transfer_block_size:
                     # update transfer and block size
                     self.ior_cmd.transfer_size.update(test[0])
                     self.ior_cmd.block_size.update(test[1])
                     # run ior
-                    self.run_ior_with_pool()
+                    self.run_ior_with_pool(plugin_path=hdf5_plugin_path)
```

Comment:
I can see the attraction in this but what happens if the test failed because of a server crash?  Avocado has a default behaviour to fail the whole script for one failure and to try circumvent this from one specific test itself seems odd.

Reply:
If it is a server crash then all the subsequent cmdlines will fail.  There is now a built in timeout for the job_manager/ior cmdline  so if the server crash causes a train wreck it should not hang  and each ior variant will timeout fairly quick.    If we are adding more and more into ior small then these new variants should have a chance to run even if one or two fail.